frontend/packages/data-portal/app/components/Breadcrumbs.tsx

```diff
@@ -44,16 +45,22 @@ function Breadcrumb({
       {text}
     </Link>
   ) : (
-    <p className={cns(className, 'font-semibold')}>{text}</p>
+    <p className={cns(className, 'font-normal')}>{text}</p>
```

Comment:
I'd be pretty hesitant about a change like this if you can avoid it. This is changing all the breadcrumbs. If you need to change this specifically for our use case, I would add a parameter or similar and leave the default as the semibold